integration-tests/infinispan-embedded/src/main/resources/resources-config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
See related comment in the native-image section of the integration test's pom. I think this file should be removed.